frontend/src/commons/Favorite.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+import { CgRemoveR } from 'react-icons/cg';
+
+export default function Favorite({ favorite, className }) {
+  return (
+    <FavoriteStyled className={className}>
+      <FavoriteHeader>
+        <LogoStyled src={favorite.crestUrl} alt="Team Logo" />
+        {favorite.name}
+        <RemoveButtonStyled>{<CgRemoveR />}</RemoveButtonStyled>
+      </FavoriteHeader>
+      <MatchStyled>
+        <div class="date">Sa 28.11.2020 15:30 Uhr</div>
+        <div>{favorite.lastMatch.homeTeam}</div>
+        <div class="result">
+          {favorite.lastMatch.homeTeamGoals} :{' '}
+          {favorite.lastMatch.awayTeamGoals}
```

Comment:
I would prefer it like this `${favorite.lastMatch.homeTeamGoals : favorite.lastMatch.awayTeamGoals}`